OpenBCI_GUI/ConsoleLog.pde

```diff
@@ -245,12 +248,12 @@ class CustomOutputStream extends PrintStream {
     try {
       File consoleDataFile = null;
       if (isWindows()) {
-        consoleDataFile = new File(sketchPath("/SavedData/Settings/"));
+        consoleDataFile = new File(sketchPath(fileDirectory));
       } else {
-        consoleDataFile = new File(sketchPath()+"/SavedData/Settings/");
+        consoleDataFile = new File(sketchPath()+fileDirectory);
```

Comment:
Question: is this if-else statement still necessary? both seem to work on windows, haven't tried other platform.